main.md

```diff
@@ -940,7 +940,7 @@ Of course, each time this happens it requires an extra protocol round trip.
 
 A more efficient manner of supplying a new nonce value is also defined --
 by including a `DPoP-Nonce` HTTP header
-in the `200 OK` response from the previous request.
+in the HTTP `200 (OK)` response from the previous request.
```

Comment:
I would suggest to only have the error code, but not the description, within the backticks.